agent_endpoint/main.py

```
@@ -0,0 +1,66 @@
+        session_path = client.session_path(
+            project=PROJECT_ID,
+            location=LOCATION_ID,
+            agent=AGENT_ID,
+            session=session_id
+        )
+
+        query_input = session.QueryInput(text=session.TextInput(text=user_message), language_code="en")
+
+        request = session.DetectIntentRequest(
+            session=session_path,
+            query_input=query_input
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)